ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/UtxoLedgerService.java

```diff
@@ -247,6 +248,30 @@ void sendUpdatedTransactionBuilder(
             @NotNull FlowSession session
     );
 
+    /**
+     * Sends transaction to counterparty whose session is open for.
+     *
+     * @param sessions The counterparty who are connected to {@link FlowSession} to send transaction.
+     * @param signedTransaction The {@link UtxoSignedTransaction} by signatories.
+     * @throws IllegalArgumentException if transaction fails verification before sending transaction.
```

Comment:
```suggestion
     * @throws CordaRuntimeException If transaction verification fails on the receiving sessions.
```

I don't see `IllegalArgumentException`s being thrown in the runtime code.